Makefile

```diff
@@ -13,7 +13,6 @@ package_macos:
 
 bundledeps_macos:
 	dylibbundler -od -b -x ./Supersonic.app/Contents/MacOS/supersonic -d ./Supersonic.app/Contents/Frameworks/ -p @executable_path/../Frameworks/
-	./copy_python_dep_osx.sh
```

Comment:
Can you create a second version of this Makefile target that removes the invocation of the .sh script, and then rename this original one to `bundledeps_macos_homebrew`? It will still be needed for building with Homebrew (more details in the comment I made over in the issue)